### PR TITLE
Healthhh

### PR DESCRIPTION
**Pull request by j013n3**

### CODE REVIEW

File: src/main/java/constants/UiConstant.java [hunk 1]
```diff
@@ -20,6 +20,9 @@ public class UiConstant {
     public static final String PARTITION_LINE = "____________________________________________________________";
     public static final String EMPTY_STRING = "";
     public static final String FULL_STOP = ".";
+    public static final String NEW_LINE = "%n";
```
Comment (L5-Z) [inline src/main/java/constants/UiConstant.java:23]: Either use `System.lineSeparator())` or we implement this as NEW_LINE and have to apply and use it across the board. Let me know your thoughts

File: src/main/java/health/HealthList.java [hunk 2]
```diff
@@ -295,7 +296,7 @@ public static void printBmiHistory() throws CustomExceptions.OutOfBounds {
         int index = 1;
         System.out.println(HealthConstant.BMI_HISTORY_HEADER);
         for (Bmi bmi : BMIS) {
-            System.out.print(index + ". ");
+            System.out.print(index + UiConstant.INDEX);
```
Comment (L5-Z) [inline src/main/java/health/HealthList.java:299]: Use `UiConstant.SPLIT_BY_PERIOD` instead and a `UiConstant.SPLIT_BY_WHITESPACE` . Naming it index can be confusing.

**Review by L5-Z — CHANGES_REQUESTED** [review]
Take a look at the suggested changes and let us know!

File: src/main/java/health/HealthList.java (hunk 2 above)
Comment (L5-Z) [reply src/main/java/health/HealthList.java:299]: Correction: It was renamed to `FULL_STOP` so do take a look again.

File: src/main/java/constants/UiConstant.java (hunk 1 above)
Comment (j013n3) [reply src/main/java/constants/UiConstant.java]: Sure

**Review by L5-Z — APPROVED** [review]
Looks good!

**Pull request merged**